nevergrad/functions/pyomo/core.py

```diff
@@ -141,9 +144,33 @@ def __init__(self, model: pyomo.Model) -> None:
         exp_tag = ",".join([n.name for n in self.all_objectives])
         exp_tag += "|" + ",".join([n.name for n in self.all_vars])
         exp_tag += "|" + ",".join([n.name for n in self.all_constraints])
-        self.register_initialization(name=exp_tag, model=self._model_instance)
+        self.register_initialization(model=self._model_instance)
+        #self.register_initialization(name=exp_tag, model=self._model_instance)
         self._descriptors.update(name=exp_tag)
```

Comment:
```suggestion
        self._descriptors.update(name=exp_tag, with_offset=bool(loss_offset))
```
Let's keep track that the function was offset, otherwise we may mix stuff that should not be compared